docker-compose.yml

```diff
@@ -130,25 +163,25 @@ services:
     image: lisk/service_transaction_statistics
     depends_on:
       nats:
-        condition: service_started
+        condition: service_healthy
       redis_persistent:
         condition: service_healthy
-      mysql:
+      mysql-primary:
         condition: service_healthy
     networks:
       - services_network
     environment:
       - SERVICE_BROKER=nats://nats:4222
       - SERVICE_STATISTICS_REDIS=redis://redis_persistent:6379/5
-      - SERVICE_STATISTICS_MYSQL=mysql://lisk:password@mysql:3306/lisk
+      - SERVICE_STATISTICS_MYSQL=mysql://lisk:password@mysql-primary:3306/lisk
```

Comment:
Can we also please support the read replicas within the transaction stats microservice?

Reply:
Applied.